### ISSUE TITLE
Way to make it dvorak/DVORAK for caps lock off/on?

### ISSUE THREAD

**Issue by andersaamodt** [ZeptByteS/dvorak-qwerty#4]
Thank you for this keyboard layout! Saved me lots of hacking & researching.

I'd like it to be all-dvorak though, with caps lock working normally. Can you point me to where in the files to edit to change this? And then I'm guessing I would run install.sh again.

**Comment by ZeptByteS** [comment]
Hi, this layout use ScrollLock to switch between Dvorak and Qwerty.
CapsLock working normally but dead-keys are replaced Qwerty symbols.

https://github.com/ZeptByteS/dvorak-qwerty/tree/ScrollLock_to_switch

**Comment by andersaamodt** [comment]
Hmm, I installed that branch and it doesn't work any different from before (after restarting & readding the Dvorak keyboard layout in Text Entry Settings).

I don't need the layout to switch to Qwerty except for Ctrl, Alt, and Super (not Scroll or Caps lock). I can switch layouts to get to Qwerty. Is there a way to get this behavior?

**Comment by ZeptByteS** [comment]
Maybe mod3 was not good for the switch.

Please try another one that use NumLock(Mod2) as switch.
https://github.com/ZeptByteS/dvorak-qwerty/tree/NumLock_to_switch

Shift-level 1 and 2 are should be preserved to Qwerty hot keys work properly. Other levels that level 3 - 8 are not reliable for modifiers. So you need another switch and Shift-levels to input text with Dvorak layout along with Qwerty hot keys. To use as switch, real-modifiers like CapsLock or NumLock are better than virtual-modifiers.

**Comment by andersaamodt** [comment]
I installed this and the behavior is the same. I need the default layout to be dvorak and caps lock to go to DVORAK. if num lock changes to qwerty that is fine; i will be using the layout switcher to go to Qwerty so i dont need a within-layout method to switch to Qwerty.

Any ideas why this NumLock_to_switch install did nothing?

**Comment by andersaamodt** [comment]
Thanks for all your help by the way! Can't wait to get this fully working

**Comment by andersaamodt** [comment]
Is there a way to have the entire layout be Dvorak, with no switching key? I can always switch to US using the language menu if I need to. (Still want Qwerty for Ctrl & Alt keyboard shortcuts, that is the whole point of using your repo!).

**Comment by eliotbaez** [comment]
I'm also wondering what is the best way to change the behavior of the caps lock key. Have you figured out how to get full Dvorak functionality yet?

**Comment by andersaamodt** [comment]
No. The layout file still works well, except that I have to have caps lock on to switch to Dvorak. Would prefer if it could be reverse so Dvorak when it's off.
